ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/desc/formatter/TextDescTableFormatter.java

```diff
@@ -390,7 +391,7 @@ private void displayAllParameters(Map<String, String> params, StringBuilder tabl
     Collections.sort(keys);
     for (String key : keys) {
       String value = params.get(key);
-      if (TABLE_IS_CTAS.equals(key)) {
+      if (TABLE_IS_CTAS.equals(key) || TABLE_IS_CTLT.equals(key)) {
```

Comment:
Does this mean that we litter the TABLE_PARAMS with this keys? Are they remaining after the table creation?

Reply:
Yes, currently that's the case. In order to have the table property available during the MetaHook#rollback, it's gotta be part of the Table object. It's just filtered out during describe commands and such but the value is there in the props. Do you think that's a dealbreaker?

Reply:
Actually, I've refactored it so that we don't persist the CTLT flag into the table props.

Reply:
Can we revert this change then?

Reply:
and even some more changes, like the formatted output and friends?